docs/IS-04 interactions.md

```diff
@@ -30,11 +40,23 @@ The API availability MUST be advertised through existing IS-04 practices inside
 }
 ```
 
-A given instance of the Configuration API MAY offer control of multiple Devices in a Node from a single URI. Alternatively there MAY be multiple instances of the API on one Node, for example, each corresponding to one Device.
-In either case, the ‘control’ endpoint for each Device’s Configuration API instance MUST be advertised, even if the URI is the same.
+**Example 2:** The 'controls' attribute of an NMOS Device of a Node which advertises a different Configuration API instance for each device
 
-This flexibility is to accommodate different relationships between Devices and Nodes. For example, some Devices may be loosely coupled to the Node, for example cards in a card frame.
-These Devices are more likely to have an instance of the API for each card.
-Others may be tightly coupled, for example a media processing pipeline on a server, where it is likely to be preferable to have one instance of the API that is advertised for each pipeline.
+```json
+...
+"controls": [
```

Comment:
I don't think this example is what IS-08 implied.

I don't think you would have multiple configuration controls advertised by each device.

I think each device will adevertise a single configuration control endpoint qualified with an api selector.

The reason for doing this is because the API web server is hosted at the Node level.

I think we need to split this example into an example with two devices each advertising a single configuration control endpoint which uses the api selector.

Reply:
Yep, good point. Or keep it to a single endpoint like the example in IS-08.